### ISSUE TITLE
Option --ivs (-i) - not working -

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#631]
**Reported by jano on 1 Jun 2009 22:03 UTC**

In Airodump-ng, the option --ivs or -i, do not work.

```
jano:~$ aircrack-ng --help | head -n 5

  Aircrack-ng 1.0 rc3 aircrack-ng/aircrack-ng@
```

```
jano:~$ sudo airodump-ng -c 11 --bssid 00:0x:xx:xx:xx:xx -i -w test_IVS wlan0
Invalid output format: IVS and PCAP format cannot be used together.
```

Thanks Jano

**Comment by aircrack-ng** [comment]
**Comment by jano on 1 Jun 2009 22:03 UTC**

I solved in this mode:
```
sudo airodump-ng -c 11 --bssid 00:0x:xx:xx:xx:xx -o ivs -w test_IVS wlan0
```
- With many new extensions/file, the option --output-format is very useful.

- But then the option --ivs, is no longer implemented ?

Jano

**Comment by aircrack-ng** [comment]
**Comment by misterx on 1 Jun 2009 22:03 UTC**

Fixed aircrack-ng/aircrack-ng@f7be66a021a2adc8ac680343ed5865b8920311cc.
